docs/process.md

```diff
@@ -41,105 +42,65 @@ Costs are
-
-The intent is that the final commit history looks something like
-
-```
-*   commit 4a1b9dd524007193213d3919d6a331109608b90c (blead)
-|\  Merge: 731a976bef ba9a3fe252
-| |
-| |     Merge implementation of foozles - RFC 1337
-| |
-| * commit ba9a3fe252d20d64d0f17968e2c47d3c4009776d
-
-...
-
-| * commit cdba169fcf0819be8f02efe3edc7aac796fb9433
-|/
-|
-|       first commit of foozles
-|
-* commit 731a976bef573afb7b94562b4b34473f6b714d5d
-```
-
-Note that there are no commits on the left side - 731a976bef is a direct parent of 4a1b9dd524. The structure is chosen as it simultaneously gives most of the benefits of squashing all the work into one commit ("this is all one logical change"), but also keeps the detail of the implementation steps, which is very useful for automatic bisection when bug hunting.
-
-See [`perlgit.pod`](https://github.com/Perl/perl5/blob/blead/pod/perlgit.pod#on-merging-and-rebasing) for how to do this with `git merge --no-ff`. This **has** to be done at the command-line - this style of merging does **not** correspond to any of the "merge" options in the GitHub UI.
-
-There is an ongoing discussion on how we decide whether to merge a PR, with the latest proposal being:
-
-* After an appropriate period, if there are not strong disagreements, and the PSC haven't rejected it, a committer will merge the PR.
-* Trivial or obviously-correct changes may be committed directly. I.e., the appropriate length is sometimes zero.
-* If objections are raised, they need to be addressed (meaning "clearly replied to", and for an RFC the reasoning recorded in "Rejected Ideas")
-* If the objection is from a subject matter expert, you need to come to an agreement.
-* If the PR stalls, ping the PSC for adjudication. We expect this will be quite rare either you should have gone through the proposal process, or it's straightforward.
-* PRs from non-committers are expected to have more scrutiny.
-
-What's an appropriate period?
-
-* For "I think this is right but want to give a chance for comments", a couple of days is probably fine.
-* For "this is a significant change that I could really use feedback on," a week or more is probably best, and the PR should probably be flagged to the list as wanting attention.
-* The later in the release cycle, the stricter we should be with ourselves.
-
-Don't panic: changes can be reverted.
+## The Process
 
+![a flowchart of the process described below](/images/flowchart.png)
+
+
+### Pre-RFC
+
+The RFC process starts with an formal proposal to add or change a language feature in Perl.  But you, the prospective author of an RFC, shouldn't start by writing one.  Start by posting to p5p that you have an idea.  Explain what problem you're solving, how you think you can solve it, and what prior art you looked at.  Be clear and concise.  Make it easy for the rest of the list to see what you're suggesting without reading an enormous wall of text, but don't lose so much detail as to be meaningless.
```

Comment:
1. done
2. "pre-rfc" is not a noun; it is the period of time before the document is being tracked by the PSC
3. the "one" is "a formal proposal", which I think is the only noun it could refer to; I will replace it with the noun
4. this seems overly prescriptive to me